mpu6050_dmp6  example does not work for ESP32 and SAMD51

mpu6050_dmp6 example does not work for ESP32
Arduino IDE 1.8.9
esp32 core 1.0.4
compile errors:
```
"D:\\arduino\\portable\\packages\\esp32\\tools\\xtensa-esp32-elf-gcc\\1.22.0-80-g6c4433a-5.2.0/bin/xtensa-esp32-elf-g++" -DESP_PLATFORM "-DMBEDTLS_CONFIG_FILE=\"mbedtls/esp_config.h\"" -DHAVE_CONFIG_H -DGCC_NOT_5_2_0=0 -DWITH_POSIX "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/config" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/app_trace" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/app_update" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/asio" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/bootloader_support" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/bt" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/coap" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/console" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/driver" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp-tls" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp32" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_adc_cal" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_event" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_http_client" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_http_server" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_https_ota" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp_ringbuf" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/ethernet" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/expat" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/fatfs" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/freemodbus" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/freertos" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/heap" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/idf_test" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/jsmn" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/json" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/libsodium" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/log" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/lwip" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/mbedtls" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/mdns" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/micro-ecc" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/mqtt" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/newlib" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/nghttp" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/nvs_flash" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/openssl" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/protobuf-c" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/protocomm" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/pthread" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/sdmmc" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/smartconfig_ack" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/soc" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/spi_flash" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/spiffs" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/tcp_transport" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/tcpip_adapter" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/ulp" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/vfs" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/wear_levelling" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/wifi_provisioning" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/wpa_supplicant" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/xtensa-debug-module" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp-face" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp32-camera" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/esp-face" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4/tools/sdk/include/fb_gfx" -std=gnu++11 -Os -g3 -Wpointer-arith -fexceptions -fstack-protector -ffunction-sections -fdata-sections -fstrict-volatile-bitfields -mlongcalls -nostdlib -w -Wno-error=maybe-uninitialized -Wno-error=unused-function -Wno-error=unused-but-set-variable -Wno-error=unused-variable -Wno-error=deprecated-declarations -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-missing-field-initializers -Wno-sign-compare -fno-rtti -MMD -c -DF_CPU=240000000L -DARDUINO=10809 -DARDUINO_FEATHER_ESP32 -DARDUINO_ARCH_ESP32 "-DARDUINO_BOARD=\"FEATHER_ESP32\"" "-DARDUINO_VARIANT=\"feather_esp32\"" -DESP32 -DCORE_DEBUG_LEVEL=0 "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4\\cores\\esp32" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4\\variants\\feather_esp32" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\esp32\\hardware\\esp32\\1.0.4\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_44490\\sketch\\MPU6050_DMP6.ino.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_44490\\sketch\\MPU6050_DMP6.ino.cpp.o"
In file included from D:\arduino\portable\sketchbook\libraries\MPU6050\examples\MPU6050_DMP6\MPU6050_DMP6.ino:51:0:

D:\arduino\portable\sketchbook\libraries\MPU6050/MPU6050_6Axis_MotionApps20.h:64:24: error: conflicting declaration 'typedef int8_t prog_int8_t'

         typedef int8_t prog_int8_t;

                        ^

In file included from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/WString.h:29:0,

                 from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/Arduino.h:146,

                 from C:\Users\hw\AppData\Local\Temp\arduino_build_44490\sketch\MPU6050_DMP6.ino.cpp:1:

D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/pgmspace.h:25:14: note: previous declaration as 'typedef char prog_int8_t'

 typedef char prog_int8_t;

              ^

In file included from D:\arduino\portable\sketchbook\libraries\MPU6050\examples\MPU6050_DMP6\MPU6050_DMP6.ino:51:0:

D:\arduino\portable\sketchbook\libraries\MPU6050/MPU6050_6Axis_MotionApps20.h:68:25: error: conflicting declaration 'typedef int32_t prog_int32_t'

         typedef int32_t prog_int32_t;

                         ^

In file included from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/WString.h:29:0,

                 from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/Arduino.h:146,

                 from C:\Users\hw\AppData\Local\Temp\arduino_build_44490\sketch\MPU6050_DMP6.ino.cpp:1:

D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/pgmspace.h:29:14: note: previous declaration as 'typedef long int prog_int32_t'

 typedef long prog_int32_t;

              ^

In file included from D:\arduino\portable\sketchbook\libraries\MPU6050\examples\MPU6050_DMP6\MPU6050_DMP6.ino:51:0:

D:\arduino\portable\sketchbook\libraries\MPU6050/MPU6050_6Axis_MotionApps20.h:69:26: error: conflicting declaration 'typedef uint32_t prog_uint32_t'

         typedef uint32_t prog_uint32_t;

                          ^

In file included from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/WString.h:29:0,

                 from D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/Arduino.h:146,

                 from C:\Users\hw\AppData\Local\Temp\arduino_build_44490\sketch\MPU6050_DMP6.ino.cpp:1:

D:\arduino\portable\packages\esp32\hardware\esp32\1.0.4\cores\esp32/pgmspace.h:30:23: note: previous declaration as 'typedef long unsigned int prog_uint32_t'

 typedef unsigned long prog_uint32_t;

                       ^

```

edit:
as to SAMD51 issues, see compiler errors listed in post  https://github.com/jrowberg/i2cdevlib/issues/500#issuecomment-616064968

for Arduino Due SAMD3X8E:

```
Arduino: 1.8.9 (Windows 7), Board: "Arduino Due (Programming Port)"

D:\arduino\arduino-builder -dump-prefs -logger=machine -hardware D:\arduino\hardware -hardware D:\arduino\portable\packages -tools D:\arduino\tools-builder -tools D:\arduino\hardware\tools\avr -tools D:\arduino\portable\packages -built-in-libraries D:\arduino\libraries -libraries D:\arduino\portable\sketchbook\libraries -fqbn=arduino:sam:arduino_due_x_dbg -ide-version=10809 -build-path C:\Users\hw\AppData\Local\Temp\arduino_build_641572 -warnings=none -prefs=build.warn_data_percentage=75 -prefs=runtime.tools.arm-none-eabi-gcc.path=D:\arduino\portable\packages\arduino\tools\arm-none-eabi-gcc\4.8.3-2014q1 -prefs=runtime.tools.arm-none-eabi-gcc-4.8.3-2014q1.path=D:\arduino\portable\packages\arduino\tools\arm-none-eabi-gcc\4.8.3-2014q1 -prefs=runtime.tools.bossac.path=D:\arduino\portable\packages\arduino\tools\bossac\1.6.1-arduino -prefs=runtime.tools.bossac-1.6.1-arduino.path=D:\arduino\portable\packages\arduino\tools\bossac\1.6.1-arduino -verbose D:\Akten\Programmierung\ArduinoProgs\DriverLibs\MPU6050-dmp6-lib_JRowberg\MPU6050_DMP6_noIRQ\MPU6050_DMP6_noIRQ.ino
D:\arduino\arduino-builder -compile -logger=machine -hardware D:\arduino\hardware -hardware D:\arduino\portable\packages -tools D:\arduino\tools-builder -tools D:\arduino\hardware\tools\avr -tools D:\arduino\portable\packages -built-in-libraries D:\arduino\libraries -libraries D:\arduino\portable\sketchbook\libraries -fqbn=arduino:sam:arduino_due_x_dbg -ide-version=10809 -build-path C:\Users\hw\AppData\Local\Temp\arduino_build_641572 -warnings=none -prefs=build.warn_data_percentage=75 -prefs=runtime.tools.arm-none-eabi-gcc.path=D:\arduino\portable\packages\arduino\tools\arm-none-eabi-gcc\4.8.3-2014q1 -prefs=runtime.tools.arm-none-eabi-gcc-4.8.3-2014q1.path=D:\arduino\portable\packages\arduino\tools\arm-none-eabi-gcc\4.8.3-2014q1 -prefs=runtime.tools.bossac.path=D:\arduino\portable\packages\arduino\tools\bossac\1.6.1-arduino -prefs=runtime.tools.bossac-1.6.1-arduino.path=D:\arduino\portable\packages\arduino\tools\bossac\1.6.1-arduino -verbose D:\Akten\Programmierung\ArduinoProgs\DriverLibs\MPU6050-dmp6-lib_JRowberg\MPU6050_DMP6_noIRQ\MPU6050_DMP6_noIRQ.ino
Using board 'arduino_due_x_dbg' from platform in folder: D:\arduino\portable\packages\arduino\hardware\sam\1.6.12
Using core 'arduino' from platform in folder: D:\arduino\portable\packages\arduino\hardware\sam\1.6.12
Build-Optionen wurden verändert, alles wird neu kompiliert
Detecting libraries used...
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev\\I2Cdev.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src\\Wire.cpp" -o nul
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\sketchbook\\libraries\\MPU6050\\MPU6050.cpp" -o nul
Generating function prototypes...
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -w -x c++ -E -CC -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\preproc\\ctags_target_for_gcc_minus_e.cpp"
"D:\\arduino\\tools-builder\\ctags\\5.8-arduino11/ctags" -u --language-force=c++ -f - --c++-kinds=svpf --fields=KSTtzns --line-directives "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\preproc\\ctags_target_for_gcc_minus_e.cpp"
Sketch wird kompiliert...
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -MMD -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\sketch\\MPU6050_DMP6_noIRQ.ino.cpp.o"
Compiling libraries...
Compiling library "I2Cdev"
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -MMD -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev\\I2Cdev.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\libraries\\I2Cdev\\I2Cdev.cpp.o"
Compiling library "Wire"
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -MMD -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src\\Wire.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\libraries\\Wire\\Wire.cpp.o"
Compiling library "MPU6050"
"D:\\arduino\\portable\\packages\\arduino\\tools\\arm-none-eabi-gcc\\4.8.3-2014q1/bin/arm-none-eabi-g++" -c -g -Os -w -std=gnu++11 -ffunction-sections -fdata-sections -nostdlib -fno-threadsafe-statics --param max-inline-insns-single=500 -fno-rtti -fno-exceptions -MMD -mcpu=cortex-m3 -mthumb -DF_CPU=84000000L -DARDUINO=10809 -DARDUINO_SAM_DUE -DARDUINO_ARCH_SAM -D__SAM3X8E__ -mthumb -DUSB_VID=0x2341 -DUSB_PID=0x003e -DUSBCON "-DUSB_MANUFACTURER=\"Arduino LLC\"" "-DUSB_PRODUCT=\"Arduino Due\"" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/libsam" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/CMSIS/Include/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\system/CMSIS/Device/ATMEL/" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\cores\\arduino" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\variants\\arduino_due_x" "-ID:\\arduino\\portable\\sketchbook\\libraries\\I2Cdev" "-ID:\\arduino\\portable\\packages\\arduino\\hardware\\sam\\1.6.12\\libraries\\Wire\\src" "-ID:\\arduino\\portable\\sketchbook\\libraries\\MPU6050" "-ID:\\arduino\\portable\\sketchbook\\libraries\\elapsedMillis" "D:\\arduino\\portable\\sketchbook\\libraries\\MPU6050\\MPU6050.cpp" -o "C:\\Users\\hw\\AppData\\Local\\Temp\\arduino_build_641572\\libraries\\MPU6050\\MPU6050.cpp.o"
D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp: In member function 'void MPU6050::PrintActiveOffsets()':

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3363:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[0], 5, 0, ",  ");

  ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3364:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[1], 5, 0, ",  ");

  ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3365:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[2], 5, 0, ",  ");

  ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3368:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[0], 5, 0, ",  ");

  ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3369:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[1], 5, 0, ",  ");

  ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3349:197: error: 'dtostrf' was not declared in this scope

 #define printfloatx(Name,Variable,Spaces,Precision,EndTxt) { Serial.print(F(Name)); {char S[(Spaces + Precision + 3)];Serial.print(F(" ")); Serial.print(dtostrf((float)Variable,Spaces,Precision ,S));}Serial.print(F(EndTxt)); }//Name,Variable,Spaces,Precision,EndTxt

                                                                                                                                                                                                     ^

D:\arduino\portable\sketchbook\libraries\MPU6050\MPU6050.cpp:3370:2: note: in expansion of macro 'printfloatx'

  printfloatx("", Data[2], 5, 0, "\n");

  ^

Bibliothek I2Cdev im Ordner: D:\arduino\portable\sketchbook\libraries\I2Cdev (legacy) wird verwendet
Bibliothek Wire in Version 1.0 im Ordner: D:\arduino\portable\packages\arduino\hardware\sam\1.6.12\libraries\Wire  wird verwendet
Bibliothek MPU6050 im Ordner: D:\arduino\portable\sketchbook\libraries\MPU6050 (legacy) wird verwendet
Bibliothek elapsedMillis in Version 1.0.5 im Ordner: D:\arduino\portable\sketchbook\libraries\elapsedMillis  wird verwendet
exit status 1
Fehler beim Kompilieren für das Board Arduino Due (Programming Port).
```

--- a/Arduino/MPU6050/MPU6050.h
+++ b/Arduino/MPU6050/MPU6050.h
@@ -44,6 +44,11 @@ THE SOFTWARE.
 
 #ifdef __AVR__
 #include <avr/pgmspace.h>
+#elif defined(ESP32)
+    #include <pgmspace.h>
+    #ifndef BUFFER_LENGTH
+        #define BUFFER_LENGTH 32
+    #endif
 #else
 //#define PROGMEM /* empty */
 //#define pgm_read_byte(x) (*(x))

--- a/Arduino/MPU6050/MPU6050_6Axis_MotionApps20.h
+++ b/Arduino/MPU6050/MPU6050_6Axis_MotionApps20.h
@@ -47,6 +47,8 @@ THE SOFTWARE.
 // http://forum.arduino.cc/index.php?topic=129407.0
 #ifdef __AVR__
     #include <avr/pgmspace.h>
+#elif defined(ESP32)
+    #include <pgmspace.h>
 #else
     // Teensy 3.0 library conditional PROGMEM code from Paul Stoffregen
     #ifndef __PGMSPACE_H_

--- a/Arduino/MPU6050/MPU6050_6Axis_MotionApps_V6_12.h
+++ b/Arduino/MPU6050/MPU6050_6Axis_MotionApps_V6_12.h
@@ -49,7 +49,9 @@ THE SOFTWARE.
 // Tom Carpenter's conditional PROGMEM code
 // http://forum.arduino.cc/index.php?topic=129407.0
 #ifdef __AVR__
-    #include <avr/pgmspace.h>
+    #include <avr/pgmspace.h>.
+#elif defined(ESP32)
+    #include <pgmspace.h>
 #else
     // Teensy 3.0 library conditional PROGMEM code from Paul Stoffregen
     #ifndef __PGMSPACE_H_

--- a/Arduino/MPU6050/MPU6050_6Axis_MotionApps_V6_12.h
+++ b/Arduino/MPU6050/MPU6050_6Axis_MotionApps_V6_12.h
@@ -49,7 +49,7 @@ THE SOFTWARE.
 // Tom Carpenter's conditional PROGMEM code
 // http://forum.arduino.cc/index.php?topic=129407.0
 #ifdef __AVR__
-    #include <avr/pgmspace.h>.
+    #include <avr/pgmspace.h>
 #elif defined(ESP32)
     #include <pgmspace.h>
 #else

--- a/Arduino/MPU6050/MPU6050_9Axis_MotionApps41.h
+++ b/Arduino/MPU6050/MPU6050_9Axis_MotionApps41.h
@@ -45,6 +45,8 @@ THE SOFTWARE.
 // http://forum.arduino.cc/index.php?topic=129407.0
 #ifdef __AVR__
     #include <avr/pgmspace.h>
+#elif defined(ESP32)
+    #include <pgmspace.h>
 #else
     // Teensy 3.0 library conditional PROGMEM code from Paul Stoffregen
     #ifndef __PGMSPACE_H_

